Dockerfile.Colab

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:18.04
+
+# Install Python 3.7
+RUN apt update
+RUN apt install software-properties-common -y
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt install python3.7 -y
+RUN python3.7 --version
```

Comment:
Does Collab support multiple Python versions? Will we have to bump this when they update Python?

Reply:
Thanks, David

As far as I know, they support 2.7, 3.6, and, 3.7. So, if someone wants to use Bytewax the only option is 3.7.

It's true that if they add a python version we will need to change our custom image as well. But only if they don't upgrade the operating system. Because if they upgrade also the operating system we won't need this special `2_27` wheel anymore and we should remove this new job.